### ISSUE TITLE
Chess Notation Display

### ISSUE THREAD

**Issue by BhagyaSRathore** [Navneet1206/Ether-Chess-OTC#4]
Show the game's move history in standard chess notation. This help players analyze the game.

**Comment by Navneet1206** [comment]
It sounds Good.

**Comment by BhagyaSRathore** [comment]
yeah!! @Navneet1206 can I start contributing on it after 31st dec?
Also, can you apply the label of SWOC

**Comment by Navneet1206** [comment]
@BhagyaSRathore  Yes of course

**Comment by BhagyaSRathore** [comment]
thank you!!

**Comment by Navneet1206** [comment]
Hey @BhagyaSRathore ,

Thank you for showing interest in contributing! Please go ahead and hit the Start button to get started on the task. Let me know if you have any questions or need any guidance along the way.

Looking forward to your contributions!

Best regards,
Navneet Vishwakarma

**Comment by BhagyaSRathore** [comment]
Sure!!!Thank you for assigning this task to me. 
can you please add a label of Swoc .

**Comment by Navneet1206** [comment]
@BhagyaSRathore  done

**Comment by Navneet1206** [comment]
Hey @yashpandav,

Thank you for showing interest in contributing! Please go ahead and hit the Star ✨ button on the repo to show your support. Let me know if you have any questions or need any guidance along the way.

Looking forward to your contributions!

Best regards,
Navneet Vishwakarma

**Comment by BhagyaSRathore** [comment]
Done

**Comment by Navneet1206** [comment]
Hey @BhagyaSRathore , 
Thank you for taking on this task.  

I have added the **SWoC** label to this issue as you asked. Please provide an update on your progress. If there are no updates within a reasonable time frame, I may need to -*close*- this issue to keep the repository organized.  

Let me know if you need any assistance or further clarification.  

Thank you!

**Comment by Navneet1206** [comment]
@BhagyaSRathore Since I haven't received any updates on this issue, I will proceed with closing it. Please feel free to reopen it if needed.

**Comment by BhagyaSRathore** [comment]
yaa! sure , but i have end sem exam in january can i start work on it after that?
